composer.json

```diff
@@ -0,0 +1,5 @@
+{
+    "require": {
+        "deployer/deployer": "^4.2"
+    }
+}
```

Comment:
How about just [downloading a phar file for deployer](https://deployer.org/download)? It should be easy to cache as well.